bookserver/config.py

```diff
@@ -81,11 +80,16 @@ def database_url(self) -> str:
     web2py_path: str = str(Path(__file__).parents[2] / "web2py")
 
     # This is the private key web2py uses for hashing passwords.
-    @property  # type: ignore
-    @lru_cache
```

Comment:
Why remove this? Otherwise, every time the code asks for the private key, it will have to open the file and read-read the data.

Reply:
Here's my rewrite that now works:

``` Python
    # This is the private key web2py uses for hashing passwords.
    @property  # type: ignore
    def web2py_private_key(self) -> str:
        @lru_cache
        def read_key():
            key_file = Path(self.web2py_path) / "applications/runestone/private/auth.key"
            with open(key_file, encoding="utf-8") as f:
                return f.read().strip()
        return read_key()
```

Reply:
this looks great, I'll include it in this PR